packages/vela/src/Vela.ts

```diff
@@ -20,14 +31,28 @@ export const options = async (
       ? { $and: [getTokenPacked(token), getOrderTypePacked(orderType)] }
       : getTokenPacked(token) || getOrderTypePacked(orderType)
 
+  const amountTpslOrder = amount
+    ? [GreaterThanOrEqual(0), GreaterThanOrEqual(0), getAmount(amount)] // needs $nth operator
```

Comment:
Need to add an $nth operator to target nth item in array (this one being the 3rd item. There is a fourth item in this array if you are wondering why not use $last)